lib/langchain/conversation.rb

```diff
@@ -58,10 +57,11 @@ def add_examples(examples)
     # @param message [String] The prompt to message the model with
     # @return [String] The response from the model
     def message(message)
-      @memory.append_user_message(message)
-      response = llm_response(message)
-      @memory.append_ai_message(response)
-      response
+      human_message = HumanMessage.new(message)
+      @memory.append_message(human_message)
+      ai_message = llm_response(human_message)
+      @memory.append_message(ai_message)
+      ai_message
```

Comment:
we now return a Message object, but because it responds to `to_s`, we get the best of both worlds